src/resources/postcss/common-responsive.pcss

```
@@ -56,94 +56,93 @@
 		}
 	}
 
+	/* From: base/full/typography/_headings.pcss */
 
 	/* -------------------------------------------------------------------------
-	 * Body 1 for --viewport-medium
+	 * Heading: h1
 	 * ------------------------------------------------------------------------- */
 
-	.tribe-common-b1--min-medium {
+	.tribe-common-h1 {
 
 		@media(--viewport-medium) {
```

ET doesn't have stylelint set up yet, but this will be flagged:

```suggestion
	.tribe-common-h1 {
		@media(--viewport-medium) {
```